packages/html-reporter/src/index.tsx

```diff
@@ -63,3 +81,35 @@ class ZipReport implements LoadedReport {
     return JSON.parse(await writer.getData());
   }
 }
+
+function mergeReports(reports: HTMLReport[]): HTMLReport {
```

Comment:
nit: I'd move this somewhere to utils, maybe make it a static function of `LoadedReport` or something.

Reply:
Moved to mergeReports.ts